src/webgpu/api/operation/command_buffer/image_copy.spec.ts

```diff
@@ -627,6 +629,250 @@ class ImageCopyTest extends GPUTest {
         unreachable();
     }
   }
+
+  async checkStencilTextureContent(
+    stencilTexture: GPUTexture,
+    stencilTextureSize: [number, number, number],
+    stencilTextureFormat: GPUTextureFormat,
+    expectedStencilTextureData: Uint8Array,
+    expectedStencilTextureDataOffset: number,
+    stencilTextureMipLevel: number
+  ): Promise<void> {
+    const copyLayout = getTextureCopyLayout('stencil8', '2d', stencilTextureSize, {
+      mipLevel: stencilTextureMipLevel,
+    });
+    assert(
+      expectedStencilTextureData.byteLength ===
+        copyLayout.mipSize[0] * copyLayout.mipSize[1] * copyLayout.mipSize[2] +
+          expectedStencilTextureDataOffset
+    );
+
+    const stencilBitCount = 8;
+
+    // Choose 'rgba8unorm' as the format of referenceTexture because WebGPU doesn't support
```

Comment:
nit: I'm a bit surprised we're doing a texture to compute the bit -> r8unorm float value, when we could just output `(1 << param.stencilBitIndex) / 255` in the shader. Am I missing something?